freebasics/templates/comments/comment.html

```diff
@@ -12,19 +11,24 @@
         {{node.user.profile.alias}}
       {% endif %}
     {% endif %}
-    </strong>
-    <span class="date">{{node.submit_date|date:"j F Y"}}</span>
-  </p>
+    <span class="comment-list__date">{{node.submit_date|date:"j F Y"}}</span>
+  </h4>
+
   {% if node.is_removed %}
-    <p>{% trans "This comment has been removed by the community. Please read our platform rules." %}</p>
+    <h5 class="comment__description">
+      {% trans "This comment has been removed by the community. Please read our platform rules." %}</h5>
   {% else %}
-    <p>{{node.comment}}</p>
-    	{% if node.parent == None %}
-        <a href="{% url 'molo.commenting:molo-comments-report' node.pk %}?next={% url 'molo.commenting:report_response' node.pk %}" class="call-to-action__text-item">{% trans "Report" %}</a>
-        {% if request.user|is_in_group:'Expert' and not node.user|is_in_group:'Expert' %}
-        <a href="{% url 'molo.commenting:molo-comments-reply' node.pk %}">{% trans "Reply"%}</a>
-        {% endif %}
+  <p>{{node.comment}}</p>
+</div>
```

Comment:
The call to action buttons are standalone elements in relation to comments list text